src/app/content/hooks/storeTextSize.ts

```diff
@@ -0,0 +1,14 @@
+import { ActionHookBody } from '../../types';
+import { setTextSize } from '../actions';
+import { textSize } from '../selectors';
+
+const hookBody: ActionHookBody<typeof setTextSize> = (services) => async() => {
```

Comment:
you can add a body to the first function level here, which is run when the app is wired up and dispatch the initial `setTextSize` there, that way everything would all be in the same place

Reply:
I tried that but redux complains about dispatching while middleware is constructing, maybe I set it up wrong though

Reply:
No you're probably right, you might need another hook in this file listening to locationChange for the setup